content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -601,7 +601,7 @@ parameters:
   ignored. If a storage account is not provided, a new storage account will be
   created in the same resource group as the cluster.
 
-#### New Azure Disk Storage Class (starting from v1.7.2)
+#### New Azure Disk StorageClass (starting from v1.7.2)
```

Comment:
For this heading, I recommend making a second change:
```suggestion
#### Azure Disk storage class (starting from v1.7.2) {#azure-disk-storage-class}
```

Reply:
It might be an idea to revise the “Azure File” heading as well, to “Azure File storage class”.